actions/websocket_actions.jsx

```diff
@@ -1412,13 +1418,30 @@ function handleCloudPaymentStatusUpdated() {
 function handleRefreshAppsBindings() {
     return (doDispatch, doGetState) => {
         const state = doGetState();
-        if (appsEnabled(state)) {
-            doDispatch(fetchAppBindings(getCurrentUserId(state), getCurrentChannelId(state)));
-        }
+        doDispatch(fetchAppBindings(getCurrentUserId(state), getCurrentChannelId(state)));
```

Comment:
@larkox Notice that I removed the `if (appsEnabled(state))` check here. This is because we don't know if the Apps plugin is available at this time, and we use this request as the check to determine this. Should we separate the concept of "the system is configured to use Apps via feature flag" and "the Apps plugin is enabled" from each other?

Merging master into this branch and resolving conflicts related to RHS bindings has brought this point up. If the frontend is reconnecting to the server, should we always try to fetch bindings, including RHS bindings? Because we don't know if the Apps plugin is available until we try to ping it. Maybe we should have a separate "ping" route that tells us if the plugin is running, so we can perform that request separately before fetching multiple sets of bindings.